app/routes/application.js

```diff
@@ -13,6 +13,8 @@ export default Route.extend({
           transition.retry();
         }
       }
+    }).catch((error) => {
```

Comment:
👍

Reply:
we could do a bit more here and should display an error on the page. or do an alert message?